code/game/objects/items/ego_weapons/aleph.dm

```diff
@@ -4,7 +4,7 @@
 	I come from the end, and I am here to stay for but a moment.\""
 	special = "This weapon has a ranged attack."
 	icon_state = "paradise"
-	force = 60
```

Comment:
This does not need a buff tbh. PL has the best healing in the game off an AOE ranged attack.

Reply:
It was worse than justitia, whereas it should be the best ALEPH, nearly on par with twilight.